### ISSUE TITLE
Remove content

### ISSUE THREAD

**Issue by lukasz-madon** [lukasz-madon/awesome-remote-job#100]
Is there content that overlaps or it's poor quality?

**Comment by tedder** [comment]
I really like having long lists of articles and companies. They get a little unwieldy (perhaps they should be at the bottom of the page?) but .. yeah. I like it.

**Comment by waldyrious** [comment]
I think every link should have a description. Even the blog posts could use a short summary. You could move the items without descriptions to a separate document, and only include in the main one those that have been properly described.

That way, the immediate result would be less content up-front, but ultimately the list would grow back to the current size, but this time with high quality, curated metadata that would justify any list size imo.

**Comment by tedder** [comment]
yeah, I'd prefer higher quality descriptions- business category, languages, home base city.

**Comment by RichardLitt** [comment]
This is a really difficult issue to answer. Could we instead ask individual contributors who added lines to add descriptions to their links, and/or add a note about quality assurance and ask people to suggest PRs _removing_ content with justification, and/or close this issue as not actionable?

**Comment by waldyrious** [comment]
I think it's fair to reach out to the people who added content without description to provide one, for a start. If we get no action in response, then it would be justified to start welcoming (or making) PRs to remove that content.

Adding a note about quality assurance to CONTRIBUTING.md (e.g. requiring that all newly added links have a description) could be done right away. I'd actually put that requirement right up at the top, perhaps at no.2 or 3.

**Comment by RichardLitt** [comment]
Great. If anyone wants to take on the legwork for this, please go ahead. :)

**Comment by fivemtebex** [comment]
Impressive project structure! The architecture really demonstrates solid design principles.
